shared/src/commonMain/kotlin/io.newm.shared/internal/HttpRoutes.kt

```diff
@@ -8,5 +8,5 @@ internal object HttpRoutes {
     private const val DEV_ENVIRONMENT = "https://garage.newm.io"
     private const val PROD_ENVIRONMENT = "https://studio.newm.io"
 
-    private const val isProduction: Boolean = true
-}
\ No newline at end of file
+    private const val isProduction: Boolean = false
```

Comment:
Can we revert this one to 'true'? That way, when we release the build to the team, they can test the connection on the main chain. Since we're doing read-only operations, we should be fine.